### ISSUE TITLE
Problems with comment in https://hyperjs.herokuapp.com/tensorflow/tiny

### ISSUE THREAD

**Issue by ToonTalk** [atanasster/hyperparameters#1]
Line 15 is 

> // fmin optmization function, retuns the loss and a STATUS_OK

Besides misspelling optimization and returns it is located far from where fmin used (line 32)

**Comment by atanasster** [comment]
Thanks for the feedback - fixed the misspelling ,as for the placement of the comment - its actually in the correct place. the `modelOpt` function is called by `fmin` and returns a status field,, as well as the `loss` value that is used to sort the best trials. Hope it makes sense

**Comment by ToonTalk** [comment]
Good. 

Line 22 also misspells optimizer

**Comment by ToonTalk** [comment]
Also line 23

> // epochs ia an integer value from 10 to 250 with a step of 5

doesn't match line 26

> epochs: hpjs.quniform(10, 30, 10),

And "is" is mistyped.

**Comment by atanasster** [comment]
thanks, got it

**Comment by atanasster** [comment]
@ToonTalk the website is now updated with the spellings fixed (https://hyperjs.herokuapp.com/tensorflow/tiny).
Thanks again for letting us know
